doc-src/ch-tar2sqfs_desc.rst

```diff
@@ -0,0 +1,51 @@
+Synopsis
+========
+
+::
+
+  $ ch-tar2sqfs TARBALL OUTDIR
+
+Description
+===========
+
+Create Charliecloud squashfs from :code:`TARBALL`  in directory :code:`OUTDIR`
+
+The resulting squashfs has a name corresponding to :code:`TARBALL` with a :code:`.sqfs` suffix
+
+Executes :code:`ch-tar2dir` and :code:`ch-dir2sqfs`
+
+Additional arguments:
+
+  :code:`--help`
+    print help and exit
+
+  :code:`--version`
+    print version and exit
+
+.. warning::
+   # TODO ask if this warning applies
+   Placing :code:`OUTDIR` on a shared file system can cause significant metadata
+   load on the file system servers. This can result in poor performance for
+   you and all your colleagues who use the same file system. Please consult
+   your site admin for a suitable location.
+
+Example
+=======
+# FIXME create example with real output
+::
+
+  $ ls -lh /var/tmp
+  total 57M
+  -rw-r-----  1 reidpr reidpr  57M Feb 13 16:14 hello.tar.gz
```

Comment:
We'll want to genericise the usernames in the final version (e.g., `charlie` is used elsewhere in the docs).

Reply:
I think in the existing docs this is inconsistent. When I'm getting real output from the new scripts I'll go through all of them so they all say `charlie`